specification/dimensions/usageunit.md

```diff
@@ -0,0 +1,93 @@
+
+Usage Unit is the unit of measure for the usage or consumption of a resource or service. Usage Unit is used in conjunction with Quantity In Usage Unit to gain understanding of usage or consumption of a given product or service in an environment.
+
+The UsageUnit column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values when the ChargeType is 'Usage'. UsageUnit should be expressed as a single unit of measure.  Units of measure used in UsageUnit SHOULD adhere to the values and format requirements specified in the Recommended Values section below.
+
+The Usage Unit for a particular line item may differ from PricingUnit when providers use different units, unit increments, or columns to determine cost. The values in UsageUnit and QuantityInUsageUnit are often listed at a finer granularity or over a different time interval than the PricingUnit and QuantityInPricingUnit. While UsageUnit may appear to relate to pricing and cost, UsageUnit is focused on resource or service consumption.  It is essential not to confuse UsageUnit with PricingUnit which is the basis for determining cost. The UsageUnit column MUST not be used as the basis for determining values related to any pricing or cost metric.
+
+## Column ID
+
+UsageUnit
+
+## Display name
+
+Usage Unit
+
+## Description
+
+The unit of measure for the usage or consumption of a resource or service
+
+## Content constraints
+
+|    Constraint   |      Value      |
+|:----------------|:----------------|
+| Column required | True            |
+| Data type       | String          |
+| Allows nulls    | True            |
+| Value format    | \<not specified> |
+
+## Recommended Values
+
+Usage units SHOULD be expressed as a single unit of measure adhering to one of the following three formats.
+
+* `<plural-units>` - "GB", "Seconds"
+* `<singular-unit> <plural-time-units>` - "GB Hours", "MB Days"
+* `<plural-units>/<singular-time-unit>` - "GB/Hour", "PB/Day"
+
+Usage units MAY be expressed with a unit quantity or time interval.  If a unit quantity or time interval is used, the unit quantity or time interval MUST be expressed as a whole number.  The following formats are valid.
+
+* `<quantity> <plural-units>` - "1000 Tokens", "1000 Characters"
+* `<plural-units>/<interval> <plural-time-units>` - "Units/3 Months"
+
+UsageUnit SHOULD be composed of the list of recommended units listed in this section unless the UsageUnit value covers a dimension not listed in the recommended unit set or if the unit covers a count-based unit distinct from recommended values in the count dimension listed in this section.  
+
+Unit names are listed in this section with the appropriate capitalization.  If the unit is not listed in the table, it is to be used over a functional equivalent with similar meaning or incompatible capitalization.  
+
+The following table and next section list the valid singular units for 3 dimensions of measurement: time, data, and count.  
```

Comment:
Should we just split the table and have 3 sub headings here (you already have two, just make a Time Units and put the values in the respective sections. Seems clearer.

Reply:
@tobrien Same comment as above; discuss soon.